arches/app/etl_modules/save.py

```diff
@@ -19,8 +19,8 @@ def save_to_tiles(userid, loadid, multiprocessing=False):
         if error_saving_tiles:
             return error_saving_tiles
 
-        _post_save_edit_log(cursor, userid, loadid, multiprocessing)
-
+        response = _post_save_edit_log(cursor, userid, loadid, multiprocessing)
+    return response
```

Comment:
Maybe better just to return `_post_save_edit_log(cursor, userid, loadid, multiprocessing)` rather than to assign the result to a variable and then return it.